apps/builder/app/builder/features/style-panel/sections/box-shadows/box-shadow-layer.tsx

```diff
@@ -49,9 +53,15 @@ const useLayer = (layer: TupleValue) => {
   }, [layer]);
 };
 
-export const Layer = (props: LayerProps) => {
+export const BoxShadowLayer = (props: LayerProps) => {
   const { index, id, layer, isHighlighted, onDeleteLayer, onLayerHide } = props;
-  const { name, shadow, color } = useLayer(layer);
+  const properties = useLayer(layer);
+
+  if (layer.type !== "tuple" || properties === undefined) {
```

Comment:
I think Box Shadow Layer should receive only layer of type Tuple so props can be changed to have a right type and do type check above. Also this looks like an error but not something what can be just `return ;`
You need to show something like `toast` that incorrect type has came to this component